mantle/kola/harness.go

```diff
@@ -1116,7 +1116,7 @@ func makeNonExclusiveTest(tests []*register.Test, flight platform.Flight) regist
 		noSSHKeyInUserdata bool
 	)
 	var flags []register.Flag
-	var nonExclusiveTestConfs []*conf.Conf
+	var nonExclusiveTestConfs []*conf.UserData
 	dependencyDirs := make(register.DepDirMap)
 	// hasExternalTestDataDir := false
 	for _, test := range tests {
```

Comment:
Unrelated: I just noticed the `NoSSHKeyIn*` logic below.  It isn't correct; tests can't be safely merged with tests that have different `NoSSHKey` flags.  Some tests expect the SSH keys to be present, and some want them absent.

Reply:
I'll add a separate PR for this. Would raising an error if `NoSSHKeyIn*` is encountered be a good course of action?

Reply:
Yeah, I think that's reasonable.